query/README.md

```diff
@@ -198,36 +66,20 @@ Literal values include numbers (integer and floating-point), quoted (both single
 | := | ieq     | Insensitive equal        | city := 'SaNtA ClArA'                                    |
 | in           | Check existence in set   | city in [‘Santa Clara’, ‘New York’] or  price in [1,2,3] |
 
-Usage of filtering features from the toolkit is similar to [sorting](#sorting).
+Note: if you decide to use toolkit provided `infoblox.api.Filtering` proto type, then you'll not be able to use [vanilla](https://github.com/grpc-ecosystem/grpc-gateway/tree/master/protoc-gen-swagger) swagger schema generation, since this plugin doesn't work with recursive nature of `infoblox.api.Filtering`.
+In this case you can use our [fork](https://github.com/infobloxopen/grpc-gateway/tree/atlas-patch/protoc-gen-swagger) which has a fix for this issue.
 
-Note: if you decide to use toolkit provided `infoblox.api.Filtering` proto type, then you'll not be able to use swagger schema generation, since it's plugin doesn't work with recursive nature of `infoblox.api.Filtering`.
+## Sorting
 
-## How to define filtering in my request?
+The syntax of REST representation of `infoblox.api.Sorting` is the following.
 
-```proto
-import github.com/infobloxopen/atlas-app-toolkit/query/collection_operators.proto;
-
-message MyRequest {
-    infoblox.api.Filtering filter = 1;
-}
-```
+| Request Parameter | Description                              | Example |
+| ----------------- |------------------------------------------| ------- |
+| _order_by         | A comma-separated list of JSON tag names. The sort direction can be specified by a suffix separated by whitespace before the tag name. The suffix “asc” sorts the data in ascending order. The suffix “desc” sorts the data in descending order. If no suffix is specified the data is sorted in ascending order. | work_address.addresss desc,first_name |
 
 ## Pagination
```

Comment:
We do not support Composite paging at the moment. Also looks like nobody uses it. What do you think about removing  it from the table in this section?

Reply:
Ok.